docs/migrating.rst

```diff
@@ -883,22 +883,28 @@ The following properties were changed to return a sequence instead of a list:
 This change should be transparent, unless you are modifying the sequence by doing things such as ``list.append``.
 
 
-Removal of ``Embed.Empty``
+Embed Changes
 ---------------------------
```

Comment:
You should remove the extraneous `-`. It only needs to match the title's length (+1 preferably)